packages/babel-traverse/test/fixtures/type-reference/input.ts

```diff
@@ -1,13 +0,0 @@
-function foo() {
```

Comment:
This test has incorrect layout (should be one-folder deeper) so it is never run. I moved it to the spread plugin.